### PR TITLE
Test & solution for #59

### PR DESCRIPTION
**Pull request by seemcat**

### CODE REVIEW

File: solutions/59.js [hunk 1]
```diff
@@ -19,6 +21,20 @@ const solution = (num) => {
   }
   return true;
 };
+
+const solution1 = (num) => {
+  if(num === 1 || num === 0 || num === -1){
+    return false;
+  }
+  for(let i = 2; i < Math.sqrt(num); i++){
```
Comment (yjlim5) [inline solutions/59.js]: why did you need to `sqrt(num)`?

**Comment by yjlim5** [comment]
Closing this PR due to merge conflicts.

File: solutions/59.js (hunk 1 above)
Comment (seemcat) [inline solutions/59.js]: screen shot 2017-06-15 at 10 43 52 am

I solved it in my own way and my code passes, why not merge?

Comment (seemcat) [inline solutions/59.js]: Ah, I see. I just saw that it has merge conflicts. I'll fix them now.
